meshroom/aliceVision/KeyframeSelection.py

```diff
@@ -158,108 +164,125 @@ class KeyframeSelection(desc.AVCommandLineNode):
+                                "flow scores.",
                     group=None,  # skip group from command line
                     enabled=lambda node: node.selectionMethod.useSmartSelection.value,
                     groupDesc=[
                         desc.FloatParam(
                             name="pxDisplacement",
                             label="Pixel Displacement",
-                            description="The percentage of pixels in the frame that need to have moved since the last keyframe to be considered for the selection.",
+                            description="The percentage of pixels in the frame that need to have "
+                                        "moved since the last keyframe to be considered for the "
+                                        "selection.",
                             value=10.0,
                             range=(0.0, 100.0, 1.0),
-                            enabled=lambda node: node.smartSelection.enabled,
+                            enabled=lambda node: node.selectionMethod.smartSelection.enabled,
                         ),
                         desc.IntParam(
                             name="minNbOutFrames",
                             label="Min Nb Output Frames",
                             description="Minimum number of frames selected to be keyframes.",
                             value=40,
                             range=(1, 100, 1),
-                            enabled=lambda node: node.smartSelection.enabled,
+                            enabled=lambda node: node.selectionMethod.smartSelection.enabled,
                         ),
                         desc.IntParam(
                             name="maxNbOutFrames",
                             label="Max Nb Output Frames",
                             description="Maximum number of frames selected to be keyframes.",
                             value=2000,
                             range=(1, 10000, 1),
-                            enabled=lambda node: node.smartSelection.enabled,
+                            enabled=lambda node: node.selectionMethod.smartSelection.enabled,
                         ),
                         desc.IntParam(
                             name="rescaledWidthSharpness",
                             label="Rescaled Frame's Width For Sharpness",
-                            description="Width, in pixels, of the frame used for the sharpness score computation after a rescale.\n"
-                                        "Aspect ratio will be preserved. No rescale will be performed if equal to 0.",
+                            description="Width, in pixels, of the frame used for the sharpness "
+                                        "score computation after a rescale.\n"
+                                        "Aspect ratio will be preserved. No rescale will be "
+                                        "performed if equal to 0.",
```

Comment:
It was not bad to have the line return when we really have it in the description.